cmd/k8s-operator/testutils_test.go

```diff
@@ -77,37 +77,28 @@ func expectedSTS(t *testing.T, cl client.Client, opts configOpts) *appsv1.Statef
 	}
 	annots := make(map[string]string)
 	var volumes []corev1.Volume
-	if opts.shouldUseDeclarativeConfig {
```

Comment:
Perhaps the `shouldUseDeclarativeConfig` option can be removed?

Reply:
Thanks, good catch